code/handlers/trackservers.q

```diff
@@ -137,13 +133,14 @@ addh:{[hpuP]
     addhw[hpuP;W]]}
 
 // return the details of the current process
+// getdetails:{(.z.f;.z.h;system"p";@[value;`.proc.procname;`];@[value;`.proc.proctype;`];@[value;(`.proc.getattributes;`);()!()])}
 getdetails:{(.z.f;.z.h;system"p";@[value;`.proc.procname;`];@[value;`.proc.proctype;`];@[value;(`.proc.getattributes;`);()!()])}
-
 / add session behind a handle
 addhw:{[hpuP;W]
     // Get the information around a process
     / info:`f`h`port`procname`proctype`attributes!(@[W;"(.z.f;.z.h;system\"p\";@[value;`.proc.procname;`];@[value;`.proc.proctype;`];@[value;(`.proc.getattributes;`);()!()])";(`;`;0Ni;`;`;()!())]);
-    info:`f`h`port`procname`proctype`attributes!(@[W;(`.servers.getdetails;`);(`;`;0Ni;`;`;()!())]);
+    //info:`f`h`port`procname`proctype`attributes!(@[W;(`.servers.getdetails;`);(`;`;0Ni;`;`;()!())]);
+    info:`f`h`port`procname`proctype`attributes!(@[W;"$[`getdetails in key`.proc;.proc.getdetails[];(.z.f;.z.h;system\"p\";`;`;$[`getattributes in key`.proc;.proc.getattributes[];()!()])]";(`;`;0Ni;`;`;()!())]);
```

Comment:
Instead of checking for existence would ``@[.proc.getdetails;`;<alt operation>]`` work instead?